Research/ResearchUI/iOS/Step View Controllers/RSDScrollingOverviewStepViewController.swift

```diff
@@ -118,6 +154,28 @@ open class RSDScrollingOverviewStepViewController: RSDOverviewStepViewController
         guard let placementType = self.imageTheme?.placementType else { return }
         let statusBarHeight = UIApplication.shared.statusBarFrame.height
         self.scrollViewBackgroundHeightConstraint.constant = (placementType == .topMarginBackground) ? statusBarHeight : CGFloat(0)
+        
+        // Update the image placement for aspect resize behavior.
+        if let imageView = self.imageView {
```

Comment:
Um... what? No. Why is this in here? Those images should be designed to allow for aspect fill. Again, this assumption breaks all overview screens that use that assumption on *both* Android and iOS.

Please don't make changes like this. If the design needs to be custom, then create a custom class to manage it.

Reply:
First, I would like you to lower the pitchfork 😈 

Let's talk about the overview screen design.  For reference in this discussion, I attached a screenshot of the tremor test for mPower 2 and a GIF of the actual UI for that task on the iPhoneSE.  

To review, on all screens where the nav header has an image view, currently, the solution is use the 1:1 aspect ratio, and utilize aspect fill, which crops out any excess.  The height of the image is controlled by the width of the screen.  This is fine on most screen designs because there isn't a lot of additional content besides a title and text copy. However, the overview screen has a lot more copy.

From what I can tell in the attached GIF, the current solution to design's request for a shorter height image on the overview screen was to auto-scroll down to the bottom of the screen, is that true?  That solution seemed unnecessary to me, and creates an outlier to the user experience, because every other screen in the app, they land viewing the top of the content.  Also, as you can tell from the GIF, if you scroll up you see 100% image which doesn't look good.  Lastly, per the screenshot of what the screen should look like, it just doesn't match the Figma design.

My solution here is to implement aspect resize content mode for the header image.  The height of the header will be resized to fit the exact aspect ratio of the image provided by design.  This will allow the the image to look good on all screen sizes while taking up the optimal vertical screen height, which is what design wants for the overview screen.

IMO, this should be an additional RSDImagePlacementType called backgroundAspectResize.  It would allow for different aspect ratio headers that would give the developers an additional tool to meet design requests. 

If we don't want to implement RSDPlacementType, we can keep this custom implementation for the overview screen.  Or, if after reading what I explained, you still disagree with my thoughts, we can push back on design.  But, I really feel like that is an unnecessary battle since we are perfectly capable of coding a solution.


<img width="179" alt="Screen Shot 2019-06-12 at 11 39 41 AM" src="https://user-images.githubusercontent.com/5590748/59365613-d4249c00-8d06-11e9-9147-1a8d8eafff23.png">

![TremorTestMpower2](https://user-images.githubusercontent.com/5590748/59365599-cec75180-8d06-11e9-860a-5610d87f51fd.gif)

Reply:
Sorry to snap, I have a lot I'm trying to manage right now. Please make changes like this in a fork as requested by @DanWebster 

Last year, this is what the design team requested. This view is used on both Motor Control and CRF. Making changes to it effects every task that uses it. The image sizing we ended up with also was a balance between what works on iPhone SE, iPhone X and is easy to implement on Android. If the images need to change for this view request new images from the design team and then test that out on all sizes, doing so for both a first run and a subsequent run.

Additionally, open the Research workspace and run the tasks in RSDCatalog.

By the way, your constraints are broken and the "Comfortable place to sit" text doesn't fit.

![Simulator Screen Shot - iPhone SE - 2019-06-13 at 09 38 22](https://user-images.githubusercontent.com/5649217/59450879-15788280-8dbf-11e9-9269-61e4a754f156.png)
![Simulator Screen Shot - iPhone SE - 2019-06-13 at 09 34 18](https://user-images.githubusercontent.com/5649217/59450880-16111900-8dbf-11e9-87c9-9ca572ad3269.png)